pkg/issuer/acme/dns/rfc2136/rfc2136.go

```diff
@@ -81,16 +81,18 @@ func NewDNSProviderCredentials(nameserver, tsigAlgorithm, tsigKeyName, tsigSecre
 	}
 	d.tsigAlgorithm = tsigAlgorithm
 
-	logf.V(logf.DebugLevel).Infof("DNSProvider nameserver:       %s\n", d.nameserver)
-	logf.V(logf.DebugLevel).Infof("            tsigAlgorithm:    %s\n", d.tsigAlgorithm)
-	logf.V(logf.DebugLevel).Infof("            tsigKeyName:      %s\n", d.tsigKeyName)
 	keyLen := len(d.tsigSecret)
 	mask := make([]rune, keyLen/2)
 	for i := range mask {
 		mask[i] = '*'
 	}
 	masked := d.tsigSecret[0:keyLen/4] + string(mask) + d.tsigSecret[keyLen/4*3:keyLen]
-	logf.V(logf.DebugLevel).Infof("            tsigSecret:       %s\n", masked)
+	logf.Log.V(logf.DebugLevel).Info("DNSProvider",
+		"nameserver", d.nameserver,
+		"tsigAlgorithm", d.tsigAlgorithm,
+		"tsigKeyName", d.tsigKeyName,
+		"tsigSecret", masked,
```

Comment:
Question, unrelated to the changes here: This is not a clear-text secret, right?

Reply:
It looks like it is a partially masked secret for some reason (the first and last 1/4ths of the secret are text and the middle half is '*').

Reply:
Thanks for helping me read out this.